tv-shows-app/src/components/feature/review/ReviewStarsInput/ReviewStarsInput.tsx

```diff
@@ -0,0 +1,17 @@
+import { StarIcon } from "@chakra-ui/icons";
+import { Flex, Text } from "@chakra-ui/react";
+
+export interface IReviewStarsInputProps {
+   label: string,
+   value: number,
+   onChange: (index : number) => void
+}
+
+export default function ReviewStarsInput({label, value, onChange} : IReviewStarsInputProps) {
+   return <Flex alignItems={'center'}>
+      <Text fontWeight={'bold'} fontSize={1} color={'white'} marginRight={0.5}>{label}</Text>
+      {
+         Array(5).fill(0).map((x, index) => {return <StarIcon key={index} boxSize='24px' color={index < value ? 'gold' : 'white'} onClick={() => {onChange(index+1)}}/>})

```

Comment:
Ovo možemo ovako:
```suggestion
         Array(5).fill(0).map((x, index) => <StarIcon key={index} boxSize='24px' color={index < value ? 'gold' : 'white'} onClick={() => {onChange?.(index+1)}} pointerEvents={onChange ? 'all' : 'none'}/>)
```

Ovako kad koristimo ovu komponentu unutar ReviewInput-a jednostavno nećemo proslijediti onChange i event, i znamo da je onda samo "prezentacijska" tj. nema nikakvu drugu funkciju osim prikazati zvijezdice.
Ako ti nije skroz jasno što sam ovdje napravio prođemo sutra :)